src/Microsoft.FeatureManagement/MicrosoftFeatureManagementFields.cs

```diff
@@ -4,14 +4,13 @@
 
 namespace Microsoft.FeatureManagement
 {
-    //
-    // Microsoft feature flag schema: https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json
-    internal static class MicrosoftFeatureFlagFields
+    internal static class MicrosoftFeatureManagementFields
     {
```

Comment:
Why remove the schema link?

Reply:
Re-added. Now the link is Microsoft Feature Management schema instead of Feature Flag schema.